packages/apidom-ls/src/config/openapi/parameter/lint/schema--mutually-exclusive.ts

```diff
@@ -2,7 +2,7 @@ import { DiagnosticSeverity } from 'vscode-languageserver-types';
 
 import ApilintCodes from '../../../codes';
 import { LinterMeta } from '../../../../apidom-language-types';
-import { OpenAPI3 } from '../../target-specs';
+import { OpenAPI30 } from '../../target-specs';
```

Comment:
Again, this should apply both for OpenAPI 3.0.x and 3.1.0 right?

```suggestion
import { OpenAPI3 } from '../../target-specs';
```